packages/kit/src/runtime/client/state.svelte.js

```diff
@@ -0,0 +1,21 @@
+/** @type {import('@sveltejs/kit').Page} */
+export const page = new (class Page {
```

Comment:
> figure out how to make it work for Svelte 4 apps

Maybe something like this works (I would hope that JS is fine with the non-existing globals in that case, as they are never accessed).

```suggestion
export const page = Number(VERSION.split('.')[0]) < 5 ? { /* ... */ } : new (class Page {
```

Reply:
Where does `VERSION` come from? We export it from `svelte/compiler` but not `svelte`

Reply:
(for now I'm doing `if ('untrack' in svelte)` but that's a shitty solution as it breaks treeshaking)

Reply:
yes it's from `svelte/compiler`, but what's stopping you from using it in the runtime? If that's all you use, the rest will be treeshaken away and all that is used is `export const VERSION = '5.1.12'`

Reply:
It's a lot of extra bundling that needs to happen. And is the compiler even treeshakeable? Even if it is, there's no guarantee it will remain so.

Right now I'm inspecting `onMount.toString()` — seems to work